style.css

```diff
@@ -0,0 +1,37 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;1,100;1,200;1,300;1,400&display=swap");
+
+* {
+  text-decoration: none;
+  color: black;
+  font-family: "Poppins", sans-serif;
```

Comment:
Awesome! By choosing a suitable font family, you did a terrific job. Please read the information below to see how the font-family parameter enables us to utilize various fonts of our choosing while using the CDN.


-    A group of fonts with comparable design elements, such as size, weight, and style, is referred to as a font family. It's crucial to select a legible typeface that matches the website's content, style, and tone when defining a font family in a web design.